### PR TITLE
add lmgtfy command

### PR DESCRIPTION
**Pull request by wnuke**
Adds one command:
`;lmgtfy <term>`: generates a lmgtfy link with the desired search term

Nice to have and easy to write.

### CODE REVIEW

File: src/main/kotlin/commands/LmgtfyCommand.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                        color = Main.Colors.SUCCESS.color
+                    }
+                }
+            }
+        }
+    }
+
+    override fun getHelpUsage(): String {
+        return "Creates a LMGTFY link with a search term:\n" +
+                "`;$name [search term]`"
```
Comment (dominikaaaa) [inline src/main/kotlin/commands/LmgtfyCommand.kt:27]: use <> instead of []

File: src/main/kotlin/commands/LmgtfyCommand.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Command
+import arg
+import doesLater
+import greedyString
+
+object LmgtfyCommand : Command("lmgtfy") {
+    init {
+        greedyString("search term") {
+            doesLater { context ->
+                // Explicit types are necessary for type inference
```
Comment (dominikaaaa) [inline src/main/kotlin/commands/LmgtfyCommand.kt:12]: unnecessary comment

File: src/main/kotlin/commands/LmgtfyCommand.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                val term: String = context arg "search term"
+                message.channel.send {
+                    embed {
+                        title = "Here you go!"
+                        description = "[Solution](https://lmgtfy.com/?q=${term.replace(" ", "+")})"
+                        color = Main.Colors.SUCCESS.color
```
Comment (dominikaaaa) [inline src/main/kotlin/commands/LmgtfyCommand.kt:18]: use Colors.BLUE, it can't really fail to parse a string so it doesn't make sense to use success

**Review by dominikaaaa — CHANGES_REQUESTED** [review]

File: src/main/kotlin/commands/LmgtfyCommand.kt (hunk 2 above)
Comment (wnuke) [reply src/main/kotlin/commands/LmgtfyCommand.kt:12]: oops, yeah that came from the example command

**Pull request merged**